scripts/sct_utils.py

```diff
@@ -1006,7 +1007,7 @@ def check_if_same_space(fname_1, fname_2):
 
 
 def printv(string, verbose=1, type='normal'):
-    """enables to print (color coded messages, depending on verbose status) 
+    """enables to print (color coded messages, depending on verbose status)
```

Comment:
Not a big deal, but you can see if you check the changes here, that you modified lines unrelated to the scope of the issue. You should have only committed what you intended to modify.

Reply:
Apologies for that mistake. I will keep that in mind for future commits.